dynamixel_controllers/src/dynamixel_controllers/joint_torque_controller.py

```diff
@@ -166,5 +170,5 @@ def process_motor_states(self, state_list):
                 self.joint_state_pub.publish(self.joint_state)
 
     def process_command(self, msg):
-        self.set_speed(msg.data)
+        self.set_goal_torque(msg.data)
```

Comment:
Looks like this is a feature, not a bug.
Step 4 of the [torque mode tutorial](http://wiki.ros.org/dynamixel_controllers/Tutorials/Creating%20a%20joint%20torque%20controller#Step_4:_Moving_the_motor) clearly states `actually move the motor we need to publish a desired **velocity** to /pan_controller/command`.

Moreover this change breaks compatibility for existing users: it is probably safer to change the name of this controller to velocity and add a new one for torque.